homeassistant/components/overkiz/sensor.py

```diff
@@ -399,6 +399,20 @@ class OverkizSensorDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfTime.SECONDS,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_BOTTOM_TANK_WATER_TEMPERATURE,
+        name="Bottom Tank Water Temperature",
```

Comment:
[It's specified](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/) that "Device, Area, and Entity names all start with a capital letter, the rest of the words are lower case":

```suggestion
        name="Bottom tank water temperature",
```

Ideally, this would use a translation key and then be translated, but I see that it's also not the case for the other sensors. So, that should be done in a another/future PR.